File: push/src/main/java/com/xiaomi/common/logger/thrift/mfs/Location.java
package com.xiaomi.common.logger.thrift.mfs;

public class Location {
    private String city;
    private String contry;
    private String province;
    private String isp;

    public void setCity(String city) {
        this.city = city;
    }

    public void setContry(String contry) {
        this.contry = contry;
    }

    public void setProvince(String province) {
        this.province = province;
    }

    public void setIsp(String isp) {
        this.isp = isp;
    }
}

File: legacy/src/main/java/com/xiaomi/common/logger/thrift/mfs/Location.kt
package com.xiaomi.common.logger.thrift.mfs

class Location {
    private var city: String? = null
    private var contry: String? = null
    private var province: String? = null
    private var isp: String? = null

    fun setCity(city: String?) {
        this.city = city
    }

    fun setContry(contry: String?) {
        this.contry = contry
    }

    fun setProvince(province: String?) {
        this.province = province
    }

    fun setIsp(isp: String?) {
        this.isp = isp
    }
}
